gui.go

```diff
@@ -121,7 +121,7 @@ func (ui *Ui) handleAddEntityToQueue() {
 		currentIndex--
 	}
 
-	if currentIndex == -1 || len(ui.currentDirectory.Entities) < currentIndex {
+	if currentIndex == -1 || len(ui.currentDirectory.Entities) <= currentIndex {
```

Comment:
Oh, this one slipped in. I could occasionally make it throw an out-of-range error by navigating around; I'm not sure what conditions were causing it, but the old code was not correct.